build.gradle.kts

```diff
@@ -28,6 +28,9 @@ kotlin {
             dependencies {
                 implementation(kotlin("stdlib"))
                 implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.1")
+                implementation("io.grpc:grpc-kotlin-stub:1.4.1") // Adjust version as needed
```

Comment:
We can't include these dependencies here since they are not android compatible. Android uses the lite versions. We'll need to figure out how to use expect functions for determining exception type and use something other than Metadata for the error details.